README.md

```diff
@@ -30,6 +30,8 @@ A curated list of resources or projects related to ox_core.
 
 - [ceeb_oxgroups](https://github.com/CeebDev/ceeb_oxgroups) - Developer resource for creating, deleting, and visualizing groups for ox_core.
 
+- [prisma](https://github.com/arlofonseca/prisma) - JavaScript/TypeScript tool containing examples of how to use Prisma ORM for FiveM.
```

Comment:
Could you maybe change the text to say "containing examples utilizing ox_core ..." to better clarify the relation to the repository?